azerothcore/azerothcore-wotlk
# fix(DB/game_event_gameobject): Brewfest gameobject in Winter Veil

## Changes Proposed:
-  Add the gobs to brewfest spawn


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4025



## Tests Performed:
- in-game




## How to Test the Changes:

1. .go ob 59177
2. see that there is no there
3. .event start 24
4. see they spawn



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
